erpnext/projects/doctype/task/task.py

```diff
@@ -43,6 +43,10 @@ def validate_dates(self):
 		if self.act_start_date and self.act_end_date and getdate(self.act_start_date) > getdate(self.act_end_date):
 			frappe.throw(_("'Actual Start Date' can not be greater than 'Actual End Date'"))
 
+		doc = frappe.get_doc("Project",self.project)
+		if  self.exp_end_date and doc.expected_end_date and getdate(self.exp_end_date) > getdate(doc.expected_end_date) :
+			frappe.throw(_("Expected end date cannot be after Project: "+doc.name+" Expected end date"))
```

Comment:
This is not the pattern we use for message, will create problem for translation.

Reply:
changed